hack/generate/samples/internal/go/memcached_with_webhooks.go

```diff
@@ -257,6 +257,7 @@ func GenerateMemcachedGoWithWebhooksSample(samplesPath string) {
 }
 
 const rbacFragment = `
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/finalizers,verbs=get;update;patch
```

Comment:
Per https://github.com/kubernetes-sigs/kubebuilder/pull/1688#issuecomment-698587502, we only need to give `update` permission for the finalizers subresource.

```suggestion
// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/finalizers,verbs=update
```

Ditto in other places in this PR.

Reply:
Done.